### PR TITLE
Earthly Lunar Page

### PR DESCRIPTION
**Pull request by avinder-red-crackle**

### CODE REVIEW

**Comment by avinder-red-crackle** [comment]
Preview URL: https://676a5f654c2ea862e87e1819--ephemeral-narwhal-3ac3b2.netlify.app/earthly-lunar

File: website/_includes/lunar/hero.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      </h1>
+
+      <p class="lg:leading-8 mt-9 lg:mt-4 text-center lg:text-left text-base lg:text-xl">
+        <span class="font-semibold">Your internal governance platform</span><br />
+        Achieve engineering excellence that is out of this world.
+      </p>
+
+      <a
+        class="bg-[#2d7e5d] inline-block mt-9 py-4 rounded-lg text-2xl text-center text-white"
+        href="https://earthly.dev/contact-us"
```
Comment (vladaionescu) [inline website/_includes/lunar/hero.html:18]:
```suggestion
        href="/contact-us"
```

**Comment by avinder-red-crackle** [comment]
Preview URL: https://676ba1844c2ea8d6b07e15cd--ephemeral-narwhal-3ac3b2.netlify.app/earthly-lunar

**Comment by avinder-red-crackle** [comment]
Preview URL: https://676d28476c5eeb7da180cfc3--ephemeral-narwhal-3ac3b2.netlify.app/earthly-lunar

File: website/_includes/lunar/governance-as-code.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      'collectors:\n- runBash: lunar collect “hasTests” true\n  hook:\n    type: before-command\n    patterns:\n    - ^go test.*\n    - ^npm test.*\n    - ^python -m .unittest.*'
+
+    const { startIndex, endIndex } = getIndicesFromOriginalText(text, document.getSelection().toString())
+    event.clipboardData.setData('text/plain', text.substring(startIndex, endIndex))
+    event.preventDefault()
+  })
+
+  document.getElementById('code-policies').addEventListener('copy', e => {
+    const text =
+      'policies:\n- name: "Should have unit tests"\n  runPython: |-\n  import lunar\n  hasTests = lunar.get(“hasTests”)\n  lunar.assertTrue(hasTests)'
```
Comment (vladaionescu) [inline website/_includes/lunar/governance-as-code.html:70]: Use correct double quotes and fix YAML indentation

```suggestion
      'policies:\n  - name: "Should have unit tests"\n  
  runPython: |-\n      import lunar\n      hasTests = lunar.get("hasTests")\n      lunar.assertTrue(hasTests)'
```

File: website/_includes/lunar/governance-as-code.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        startIndex = checkSubsequentChars(startIndex)
+      }
+    }
+
+    return { startIndex, endIndex }
+  }
+
+  document.getElementById('code-collectors').addEventListener('copy', e => {
+    const text =
+      'collectors:\n- runBash: lunar collect “hasTests” true\n  hook:\n    type: before-command\n    patterns:\n    - ^go test.*\n    - ^npm test.*\n    - ^python -m .unittest.*'
```
Comment (vladaionescu) [inline website/_includes/lunar/governance-as-code.html:61]: Use correct double quotes and fix YAML indentation

```suggestion
      'collectors:\n  - runBash: lunar collect "hasTests" true\n    hook:\n      type: before-command\n    patterns:\n        - ^go test.*\n        - ^npm test.*\n    
    - ^python -m .unittest.*'
```

**Comment by avinder-red-crackle** [comment]
Preview URL: https://676e3d8cb0b774a09482072c--ephemeral-narwhal-3ac3b2.netlify.app/earthly-lunar

**Comment by avinder-red-crackle** [comment]
Preview URL: https://67723ee493a902f9540f22a0--ephemeral-narwhal-3ac3b2.netlify.app/earthly-lunar

**Review by brandonSc — APPROVED** [review]
Looks great! 👍

**Pull request merged**